### PR TITLE
Avoid Libliftoff timeout

### PR DESCRIPTION
**Pull request by brain-anti-freeze**
To address #1298

### CODE REVIEW

File: src/drm.cpp [hunk 1]
```diff
@@ -443,6 +443,7 @@ struct drm_t {
 
 	struct liftoff_device *lo_device;
 	struct liftoff_output *lo_output;
+	struct liftoff_output_apply_options *lo_options;
```
Comment (emersion) [inline src/drm.cpp:446]: I don't think we need to store a pointer to the options? Can't we just declare the options right before calling `liftoff_output_apply()`?

Comment (Joshua-Ashton) [reply src/drm.cpp:446]: Yeah

Comment (brain-anti-freeze) [reply src/drm.cpp]: Done

**Review by emersion — APPROVED** [review]
Thanks!

**Pull request merged**